### PR TITLE
feat: Create top page

### LINKED ISSUE #2: feat: Create top page
connect to server to display live data
display weather API data

### PR DESCRIPTION
**Pull request by marimarimarietty**

### CODE REVIEW

File: src/components/organisms/Livedata.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+import { Doughnut } from "react-chartjs-2";
+
+type Props = {
+  currTemp: number;
+  currHumid: number;
+};
+
+export default function Livedata(props: Props) {
+  const [tempColor, setTempColor] = useState<any>("rgb(255, 99, 132)");
+  const [humidColor, setHumidColor] = useState<any>("rgb(255, 99, 132)");
```
Comment (Takashicc) [inline src/components/organisms/Livedata.tsx]: Change the `<any>` to `<CanvasGradient | "rgb(255, 99, 132)">`

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
+import "@/styles/reset.scss";
 import "@/styles/globals.scss";
+import "@/styles/responsive.scss";
 import type { AppProps } from "next/app";
 
 function MyApp({ Component, pageProps }: AppProps) {
```
Comment (Takashicc) [inline src/pages/_app.tsx:6]: Are you using this file?

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,72 +1,14 @@
...
-            <h2>Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
+export default function Index() {
+  const pageid = "index";
```
Comment (Takashicc) [inline src/pages/index.tsx:7]: Are you using this file?

**Review by Takashicc — CHANGES_REQUESTED** [review]

File: src/pages/_app.tsx (hunk 2 above)
Comment (marimarimarietty) [reply src/pages/_app.tsx:6]: Yes, I need these files.

File: src/pages/index.tsx (hunk 3 above)
Comment (marimarimarietty) [reply src/pages/index.tsx:7]: This is top page and I need this file.

**Review by Takashicc — APPROVED** [review]
LGTM!

**Pull request merged**